### PR TITLE
Fix issues on applications module

### LINKED ISSUE #1748: Issues on applications module
### Expected behavior

Interoperability features should work as defined by product specs.

### Actual behavior

Since most backend interoperability issues were solved, new issues on mobile appeared (we stopped using mocks and started consuming real live data from Lisk Service).

The main issues found are:

1. Application details screen not rendering properly the data of each application. Most data is "Not available", and fields such as dates are being parsed wrongly. This is because queries are making wrong requests.

3. Non-active applications are being listed on applications exploring and able to add and bookmark.

5. No default icon for transactions. Also missing the "interoperability transactions" icon.

7. Missing title for interoperability transactions on transaction details screen.

9. Switching applications break token list fetching.
Uploading Simulator Screen Recording - iPhone 14 - 2023-04-05 at 16.12.08.mp4…

### Which version(s) does this affect? (Environment, OS, etc...)
All.

### PR DESCRIPTION
**Pull request by ClementeSerrano**
### What was the problem?

This PR resolves #1748 and #1777

### How was it solved?

- [x] Fix application details screen not rendering properly the data of each application. Most data is "Not available", and fields such as dates are being parsed wrongly. 
- [x] Fix unregistered applications are being listed on applications exploring and able to add and bookmark. 
- [x] Fix no default icon for transactions. Also missing the "interoperability transactions" icon.
- [x] Fix missing title for interoperability transactions on transaction details screen.
- [x] Fix switching applications break token list fetching.
- [x] Fix application stats changes crashes the app

### How was it tested?

- [x] iOS Simulator.
- [x] Android Emulator.

### CODE REVIEW

**Review by ManuGowda — COMMENTED** [review]
Screenshot 2023-04-18 at 12 21 46 PM

- Back button not visible
- `lisk.com` link is not clickable

**Comment by ManuGowda** [comment]
Screenshot 2023-04-18 at 12 22 54 PM
pinning not working?

File: src/modules/Accounts/components/TokenList/TokenList.js [hunk 1]
```diff
@@ -117,6 +117,7 @@ export default function TokenList({ mode = 'overview', address, style }) {
             fetchNextPage={fetchNextTokensPage}
             hasNextPage={mode === 'full' && hasTokensNextPage}
             isFetchingNextPage={isFetchingTokensNextPage}
+            ItemSeparatorComponent={() => <View style={{ height: 8 }} />}
           />
         )}
         renderLoading={() => <TokenListSkeleton />}
```
[the remark this answers is not in the document]
Comment (ClementeSerrano) [reply src/modules/Accounts/components/TokenList/TokenList.js:123]: yes we need. `DataRenderer` receives a function that then invoques, not a react element.

File: src/utilities/helpers.js [hunk 2]
```diff
@@ -226,3 +226,20 @@ export function spliceArray(arr, start, deleteCount, ...addItem) {
...
+
+/**
+ * Adds a string to an array without repeating it and returns a new array.
+ *
+ * @param {string[]} array - The input array.
+ * @param {string} string - The string to be added.
+ * @returns {string[]} A new array with the added string, if it was not already present.
+ */
+export function addUniqueStringToArray(array, string) {
+  const newArray = [...array];
```
[the remark this answers is not in the document]
Comment (ClementeSerrano) [reply src/utilities/helpers.js]: done

**Review by ManuGowda — COMMENTED** [review]
Screenshot 2023-04-19 at 11 12 55 AM


The values are not converted to their base denominations

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**